sdk/src/connections/target/adsd3500_sensor.cpp

```diff
@@ -455,6 +455,10 @@ aditof::Status Adsd3500Sensor::stop() {
 aditof::Status Adsd3500Sensor::getAvailableModes(std::vector<uint8_t> &modes) {
     modes.clear();
     for (const auto &availableMode : m_availableModes) {
+#ifndef ENABLE_PCM
+        if (availableMode.isPCM)
+            continue;
+#endif
```

Comment:
In case `ENABLE_PCM` is set to `OFF`, shouldn't this for filter out the `PCM mode`?